RELEASING.md

```diff
@@ -2,14 +2,31 @@
 
 ## Main Release
 
-The following steps should be followed to release the main version:
- 1. Run the workflow [Prepare Release Main Version](https://github.com/zaproxy/zaproxy/actions/workflows/prepare-release-main-version.yml),
-    to prepare the release. It creates a pull request updating the version and indicating the remaining release tasks;
- 2. Finish the tasks indicated in the pull request;
- 3. Merge the pull request, to create the tag and the draft release (done by
-    [Release Main Version](https://github.com/zaproxy/zaproxy/actions/workflows/release-main-version.yml));
- 4. Verify the draft release;
- 5. Publish the release.
+### Pre release
+
+These tasks use checkboxes so that they can be copied into an issue.
+
+- [ ] Update dependencies - these can be checked using the [zaproxy](https://github.com/zaproxy/zaproxy) `dependencyUpdates` gradle task
+- [ ] Update the [gettingStarted](https://github.com/zaproxy/zap-extensions/tree/main/addOns/gettingStarted) ODT document and regenerate the PDF.
+- [ ] Update the [MacOS JRE](https://github.com/zaproxy/zaproxy/blob/main/buildSrc/src/main/kotlin/org/zaproxy/zap/distributions.gradle.kts).
+- [ ] Update the [MacOS Coyyright statement](https://github.com/zaproxy/zaproxy/blob/main/zap/src/main/macOS/OWASP%20ZAP.app/Contents/Info.plist).
+- [ ] Update [Constant](https://github.com/zaproxy/zaproxy/blob/main/zap/src/main/java/org/parosproxy/paros/Constant.java)#VERSION_TAG.
+- [ ] Add and use a [Constant](https://github.com/zaproxy/zaproxy/blob/main/zap/src/main/java/org/parosproxy/paros/Constant.java).upgradeFrom`<version>`() method.
+- [ ] Create help release page
+  - Development / bug fix issue links can be generated using the [zap-admin](https://github.com/zaproxy/zap-admin) `generateReleaseNotes` task.
+  - Library changes can be determined by diffing [LEGALNOTICE.md](https://github.com/zaproxy/zaproxy/blob/main/LEGALNOTICE.md) with the version at the previous release.
+- [ ] Create the [zap-admin](https://github.com/zaproxy/zap-admin) version and news files
+- [ ] Prepare blog post
+
+### Release Process
+
+The following steps should be followed in the given order to release the main version:
+- [ ] Run the workflow [Prepare Release Main Version](https://github.com/zaproxy/zaproxy/actions/workflows/prepare-release-main-version.yml), to prepare the release. It creates a pull request updating the version and indicating the remaining release tasks;
+- [ ] Finish the tasks indicated in the pull request;
```

Comment:
Add before, e.g.:
`- [ ] Update [SECURITY.md](https://github.com/zaproxy/zaproxy/blob/main/SECURITY.md) to mention the latest version.`

Reply:
Tweaked the existing issue, to list these tasks.